Source/Particles/Pusher/CopyParticleAttribs.H

```diff
@@ -39,25 +39,27 @@ struct CopyParticleAttribs
     amrex::ParticleReal* AMREX_RESTRICT uypold = nullptr;
     amrex::ParticleReal* AMREX_RESTRICT uzpold = nullptr;
 
-    CopyParticleAttribs (const WarpXParIter& a_pti, TmpParticles& tmp_particle_data) noexcept
+    CopyParticleAttribs (const WarpXParIter& a_pti, TmpParticles& tmp_particle_data,
```

Comment:
Can you please add a doxygen string here?
`a_offset` is selecting an attribute by passing an offset?

The param block in the struct doxygen
```
 * \param a_pti iterator to the tile containing the macroparticles
 * \param a_tmp holder for the temporary particle data
```
should be moved to the constructor doc string here :)

Reply:
Yes, added. `a_offset` is selecting the particle to start out. This is only non-zero when we use the field gather buffers. There, we partition the particles in the arrays and do the field gather + push operation separately on the different parts. So we don't always want to start at 0.